lib/iris/fileformats/pp.py

```diff
@@ -163,10 +190,37 @@
     ("lbproj", (31,)),
     ("lbtyp", (32,)),
     ("lblev", (33,)),
-    ("lbrsvd", (34, 35, 36, 37,)),
+    (
+        "lbrsvd",
+        (
+            34,
+            35,
+            36,
+            37,
+        ),
+    ),
     ("lbsrce", (38,)),
-    ("lbuser", (39, 40, 41, 42, 43, 44, 45,)),
-    ("brsvd", (46, 47, 48, 49,)),
+    (
+        "lbuser",
+        (
+            39,
+            40,
+            41,
+            42,
+            43,
+            44,
+            45,
+        ),
+    ),
+    (
+        "brsvd",
+        (
+            46,
+            47,
+            48,
+            49,
+        ),
+    ),
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?